A01/COMP348-A1-AlexJeanFonz/src/main/java/com/alexjeanfonz/Program.java

```diff
@@ -1,12 +1,58 @@
 package com.alexjeanfonz;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 /**
  * App entrypoint
  * @author jeanrobatto
  */
 public class Program {
     
     public static void main(String[] args) {
+        PartOneTests();
+        
+        Employee[] employees = new Employee[0];
+        try {
+            employees = ReadEmployeesFromFile();
+        } catch (Exception e) {
+            System.out.println(e.toString());
+        }
+        
+        // TEST: Output the first read employee
+        if (employees.length > 0) {
+            System.out.println(employees[0].toString());
+        } else {
+            System.out.println("Employees list is empty!");
+        }
+        
+    }
+    
+    public static Employee[] ReadEmployeesFromFile() {
```

Comment:
Documentation missing! and in Java (as opposed to C#) the convention for method names is lower camel case: readEmployeesFromFile